reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -209,11 +297,11 @@ class StaticHtmlReporter : Reporter {
 
         h2 {
             id = "rule-violation-summary"
-            +"Rule Violation Summary ($errorCount errors, $warningCount warnings, $hintCount hints to resolve)"
+            +"Policy Violations (errors: $errorCount, warnings: $warningCount, hints: $hintCount)"
```

Comment:
Hm, what's the reason for changing terminology from `rule` to `policy` violations.
I believe it makes things inconsistent with other reports and how it's being called in the code currently. The latter I'd see not as an issue because not everything needs to be consistent with the UI.

Reply:
The point is that "policy violation" is clearer, a "rule violation" could just be any error. In any case, let's leave terminology alignment to another PR.